lib/evmone/tracing.cpp

```diff
@@ -149,6 +150,60 @@ class InstructionTracer : public Tracer
         m_out << std::dec;  // Set number formatting to dec, JSON does not support other forms.
     }
 };
+
+
+class StandardTracer : public InstructionTracer
```

Comment:
Standard according to what? Is this matching some other EIP? Please add a description to clarify.

Reply:
Added a doc comment